### PR TITLE
[networking_mapper] Gather facts only on needed hosts

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:

### CODE REVIEW

**Comment by morucci** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by morucci** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Review by rebtoor — APPROVED** [review]

File: roles/networking_mapper/molecule/default/converge.yml [hunk 1]
```diff
@@ -132,6 +132,11 @@
           }}
       loop: "{{ _inventory_host_items }}"
 
+    - name: Prepare the list of relevant instances
```
Comment (cjeanner) [inline roles/networking_mapper/molecule/default/converge.yml:135]: keeping this for future debugging?

**Comment by cjeanner** [comment]
/approve

tested in my pile of changes, works fine, thanks!

**Comment by cjeanner** [comment]
recheck

**Pull request merged**